python/ray/autoscaler/aws/node_provider.py

```diff
@@ -169,6 +169,13 @@ def set_node_tags(self, node_id, tags):
     def create_node(self, node_config, tags, count):
         tags = to_aws_format(tags)
         conf = node_config.copy()
+
+        # Delete unsupported keys from the node config
+        try:
+            del conf["Resources"]
```

Comment:
why this key in particular?

Reply:
It's added to the config in order for `request_cores` to know how many workers to start.

Scripts / remote functions shouldn't care about the cluster topology; they just care about resources.

In the future it might be possible to extend this so that there are multiple types of `worker_node` e.g. one might have CPU resources and another GPU. You could then dynamically request a GPU for a task and have it spin down when done.